bot/index.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+import { Rettiwt } from 'rettiwt-api';
+import 'dotenv/config';
+import 'axios';
+
+const users = [
+    { username: '5m5v_fr', id: '1169895168259964933', lastSentTweetId: null },
+    { username: '5m5v_de', id: '1169896309144207360', lastSentTweetId: null },
+];
```

Comment:
This list is temporary - I'd like to recreate all of our accounts to start from a clean slate (right now, we lost the password to half of them and the English one has been banned).